examples/latent_sde.py

```diff
@@ -342,26 +390,24 @@ def main():
     parser.add_argument('--rtol', type=float, default=1e-3)
     parser.add_argument('--atol', type=float, default=1e-3)
 
-    parser.add_argument('--show-prior', type=utils.str2bool, default=True)
-    parser.add_argument('--show-samples', type=utils.str2bool, default=True)
-    parser.add_argument('--show-percentiles', type=utils.str2bool, default=True)
-    parser.add_argument('--show-arrows', type=utils.str2bool, default=True)
-    parser.add_argument('--show-mean', type=utils.str2bool, default=False)
-    parser.add_argument('--hide-ticks', type=utils.str2bool, default=False)
+    parser.add_argument('--show-prior', type=str2bool, default=True)
+    parser.add_argument('--show-samples', type=str2bool, default=True)
+    parser.add_argument('--show-percentiles', type=str2bool, default=True)
+    parser.add_argument('--show-arrows', type=str2bool, default=True)
+    parser.add_argument('--show-mean', type=str2bool, default=False)
+    parser.add_argument('--hide-ticks', type=str2bool, default=False)
```

Comment:
What's the rationale for not using "store_true/false"?

Reply:
It makes life easier when I need to have small sweeps using shell scripts, e.g. I could now just do 
```shell script
for arg in True False; do 
... 
done
```
I just updated it so that now it also works for arguments with no value passed in, e.g. both `--adjoint` and `--adjoint True` work.